### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'get_screenshot'

### ISSUE THREAD

**Issue by princessGasmask** [Pompeiro/TFT-DSS#5]
![cmd_9HIk2DAlkw](https://user-images.githubusercontent.com/5853862/96538593-7c813380-125e-11eb-95de-c69b265050df.png)

I keep getting AttributeError: 'NoneType' object has no attribute 'get_screenshot' on each run. I've installed all the dependencies and I'm running on Win 10.

**Comment by Pompeiro** [comment]
@princessGasmask 
Sorry for delay.
Going back to work on project soon.
I think answer for ur problem is that u need to check lines 180 and 182 in code, while u want to test it live when u are playing TFT then:
wincap = WindowCapture('League of Legends (TM) Client')

# wincap = None
otherwise u are in "sandbox" mode when u dont need to have game opened, but its for test purposes, so if you just want test that OCR works correctly then make screenshot from TFT while u are playing and you have 5 champions to buy and name it ss.jpg and put it in the same directory that TFTDSS.py is. You need to be in windowed mode in game.

tl;dr
uncomment 180 then comment line 182 like upper.
Run TFT game in windowed mode
Run TFTDSS.py
While u have champions to buy then click scan&go for most utility coverage under one button.

There is no manual so you need to try it yourself or asking me.

IF u can then let me know that helped and u can see points for champions

**Comment by xosmi** [comment]
Can confirm, was running into the same issue until i un-commented the line at 180. works fine after that.
I don't know much about TFT, only ever having 'played' it to get mission rewards.
Still, just buying the recommended ones and just lining them up and slapping items i gained on whomever still got me 3rd place on my 1st try.

![tftdss](https://user-images.githubusercontent.com/49854758/97080742-fcf8b900-15fd-11eb-8763-a7f930e930b7.png)

**Comment by Pompeiro** [comment]
@xosmi thank you for confrimation, I am happy that someone launched my software.

**Comment by princessGasmask** [comment]
it still didn't work for me. I looks like it's working but it only screencaps a small portion of the screen. The console also gives me some type of error. I decided to start with a fresh pull of the source. I'll update again if I have another error.

**Comment by princessGasmask** [comment]
UPDATE: It worked this time. Just to confirm: Blue is the optimal move right?

**Comment by Pompeiro** [comment]
@princessGasmask are u in windowed mode?

**Comment by Pompeiro** [comment]
You need to be in windowed mode, blue is between green(best) and red(worst). When I am using this program then  I am only working on tkinter GUI where u have points after scan&go button.

**Comment by princessGasmask** [comment]
Perfect! I got it working so I'll close this issue now. Thanks for the help!
